bin/GameIndex.yaml

```diff
@@ -7052,6 +7061,20 @@ SLES-50072:
   name: "Street Fighter EX3"
   region: "PAL-E"
   compat: 5
+  patches:
+    5E13E6D6:
+      content: |-
+        author=kozarovv
+        // Fix for missing effects in Hardware mode.
```

Comment:
I'm not keen on adding patches for HW mode.

Reply:
Nor am I, because it will ruin the experience in software mode.

Reply:
Understandable. Just in case, this patch is not removing any effect. This make missing effects to appear in HW mode, without affecting SW mode. Is similar patch (same issue, different patch approach) to Galerians Ash patch that is in game DB already.

Reply:
Tons of other patches weren't added either due to removing effects or hacky workarounds.

Reply:
```
SLES-51343:
  name: "Galerians - Ash"
  region: "PAL-M5"
  patches:
    1AE08CF5:
      content: |-
        author=kozarovv
        // Floats workaround to make HW GSDX mode working.
        // There is still overflow on GS, but this time GSDX can handle it just fine in HW mode.
        patch=1,EE,001e7130,word,3c017e00
        patch=1,EE,001e7138,word,44812800
        patch=1,EE,001e7150,word,44813000
        patch=1,EE,001e7160,word,44813800
        patch=1,EE,001e7170,word,44810000
```
So if it doesn't touch the SW experience at all, isn't it fine to have it included?

Reply:
`Is similar patch (same issue, different patch approach) to Galerians Ash patch that is in game DB already.`
I see
https://github.com/PCSX2/pcsx2/commit/15dfad0ebc3d8053dc4a521a0778aa4e9b41e44b

Personally not a fan, perhaps @tellowkrinkle would like to take a look at it #1684?

Reply:
Okay, i removed Street Fighter EX3. Something that needs to be looked at in ETA unknown period.